capstone/src/main/webapp/script.js

```diff
@@ -277,24 +277,13 @@ function loadResponseChart(totalResponses, precinct) {
   chart.draw(stats, null);
 }
 
-function mapSentiment(colorMap) {
-  const precinctLayer = mapAndSelection.map;
-  for (const [precinctName, precinctColor] of colorMap) {
-    precinctLayer.revertStyle();
-    precinctLayer.setStyle({fillColor: '#CECDBC',
-      fillOpacity: 0.9});
-    precinctLayer.overrideStyle(precinctLayer.getFeatureById(
-        searchPrecinctsByStation(precinctName)),
-    {fillColor: precinctColor, fillOpacity: 0.9});
-  }
-}
-
 function searchPrecinctsByStation(desiredStation) {
-  mapAndSelection.map.forEachFeature( function(feature) {
-    if (feature.getProperty('station') == desiredStation) {
-      return feature.getProperty('id');
+    let precinctID = [200,300,400,500,600,800,900,1000];
+  for (let i = 0; i < precinctID.length; i++) {
```

Comment:
You could do a for each loop here since you are iterating element-by-element from the start of the list to the end.

Reply:
I can't get for each loops to work in javascript. It's normally precinctID.forEach(function), but it wasn't working how I wanted it to.

Reply:
I've decided to just do for loops since I've been having trouble getting the for each to work.